### DISCUSSION TITLE
Select Options save button is greyed out on v11.x

### DISCUSSION THREAD

**Discussion by EliShklovsky** [pimcore/pimcore#17503] [Q&A / Support] [answered]
Hi
On all my clean installations the "Select Options" save button is greyed out.
The same is on Pimcore's [official demo](https://demo.pimcore.fun/admin/)
And also on Pimcore's [portal demo.](https://portal-demo.pimcore.com/admin/)
Any idea?
![SelectOptions-Save-is greyed](https://github.com/user-attachments/assets/5be10f91-e354-4c46-80a8-da89746a93ae)

**Comment by kingjia90** [comment]
Hello Eli,

that looks like when the config write setting is set to either `disabled`, or `symfony-config` 
https://github.com/pimcore/pimcore/blob/dd1b1f8e50ac9c8270454d384bc20cbb7c1ad699/doc/21_Deployment/03_Configuration_Environments.md?plain=1#L94-L98
but the yaml file is not writable (please also check unix permission, owner etc..).

**Reply by kingjia90** [reply]
More technically please see if you are under these circumstances in the case of select_options
https://github.com/pimcore/pimcore/blob/a7f3fef2f8368122cb69c75bd1d4bb6726e638cc/lib/Config/LocationAwareConfigRepository.php#L132-L140

**Reply by EliShklovsky** [reply]
Hi @kingjia90 
Thanks for you answer, but I rather use the settings-store:

```
select_options:
            write_target:
                type: 'settings-store'
```

Now the button is green, and when I set the name of the option I get this error:

> URL: /admin/class/select-options-get
Message: Not Found
Trace: 
in /opt/bitnami/pimcore/vendor/pimcore/admin-ui-classic-bundle/src/Controller/Admin/DataObject/ClassController.php:2085
#0 /opt/bitnami/pimcore/vendor/pimcore/admin-ui-classic-bundle/src/Controller/Admin/DataObject/ClassController.php(1962): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\ClassController->getSelectOptionsConfig()
#1 /opt/bitnami/pimcore/vendor/symfony/http-kernel/HttpKernel.php(181): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\ClassController->selectOptionsGetAction()
#2 /opt/bitnami/pimcore/vendor/symfony/http-kernel/HttpKernel.php(76): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#3 /opt/bitnami/pimcore/vendor/symfony/http-kernel/Kernel.php(197): Symfony\Component\HttpKernel\HttpKernel->handle()
#4 /opt/bitnami/pimcore/public/index_test.php(38): Symfony\Component\HttpKernel\Kernel->handle()
#5 {main}

**Reply by kingjia90** [reply]
probably would need to change `read_target` as well
https://github.com/pimcore/pimcore/blob/ba7de58efb4f9a3062e343d6bd2474d7a8f7ce5d/bundles/CoreBundle/config/pimcore/default.yaml#L343-L346

**Reply by EliShklovsky** [reply, marked as answer]
Hi @kingjia90 
It is now working, thanks!

```css
        select_options:
            write_target:
                type: 'settings-store'
            read_target: 
                type: 'settings-store'
```
